src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/services/AttendanceServiceTest.kt

```diff
@@ -769,6 +769,34 @@ class AttendanceServiceTest {
                 .putCaseNoteAmendment(1, 1, "IEP rescinded: attended")
     }
 
+    @Test
+    fun `should post case note amendment going from paid attendance to unpaid absent refused`() {
```

Comment:
I think this tests the scenario:
  -> unpaid (IEP Warning) -> paid attendance (IEP rescinded) -> unpiad (IEP warning reinstated)